src/Api/Modules/PotStageHistory.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Zoho\CRM\Api\Modules;
+
+class PotStageHistory extends AbstractRecordsModule
+{
+    protected static $primary_key = 'POTENTIALSTAGEHISTORYID';
+
+    protected static $associated_entity = \Zoho\CRM\Entities\PotStageHistory::class;
+
+    protected static $supported_methods = [
+        'getRelatedRecords',
+    ];
+
+    public function getPotentialHistory($potentialId)
+    {
+        return $this->getRelatedById('Potential', $potentialId)->fetch()->getContent();
```

Comment:
You are setting the `parentModule` (1st argument) of the request to `'Potential'`. But the documentation says it must be `'Potentials'` (plural), so even if it works either way, you must change it.